src/hyperlight_wasm/src/sandbox/loaded_wasm_sandbox.rs

```diff
@@ -53,6 +53,12 @@ impl LoadedWasmSandbox {
     /// On success, return an `Ok` with the return
     /// value and a new copy of `Self` suitable for further use. On failure,
     /// return an appropriate `Err`.
+    ///
+    /// # Errors
+    ///
+    /// Returns `Err(HyperlightError::PoisonedSandbox)` if the sandbox is in a
```

Comment:
While correct, there reason the sandbox was poisoned was because of another error that occured. Maybe it's worth clarifying, since it's possible to interpret this as "The sandbox is not poisoned until it returns Err(HyperlightError::PoisonedSandbox)"